roles/lib_openshift/library/oc_adm_router.py

```diff
@@ -205,31 +205,31 @@
     aliases: []
 author:
 - "Kenny Woodson <kwoodson@redhat.com>"
-extends_documentation_fragment:
-- There are some exceptions to note when doing the idempotency in this module.
```

Comment:
@kwoodson I wasn't quite sure what to do with this content. I tried adding it to the description, but the description field is automatically formatted and wasn't playing nice with the current way of listing the exceptions.

`extends_documentation_fragment` is used for adding additional documentation to an existing doc, not for providing additional content for the current doc.